src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityRefProvEntryViewImpl.java

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.web.client.widget.entity.controller;
+
+import org.gwtbootstrap3.client.ui.Anchor;
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.html.Text;
+import org.sagebionetworks.web.client.utils.Callback;
+
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class EntityRefProvEntryViewImpl extends Composite implements EntityRefProvEntryView {
```

Comment:
Please remove "extends Composite".  Member variable "widget" is the UI component for this view.